### ISSUE TITLE
Requête 

### ISSUE THREAD

**Issue by jeff44850** [Snille/MMM-Videoplayer#1]
Bonjour 
Tout d'abord merci pour ce module depuis le temps que je cherche a diffuser automatiquement une vidéo local !!!
J'ai cependant une requête sur l'évolution de ce module 
Serait il possible de lire un ensemble de vidéos provenant d'un dossier ?
Cordialement
Jeff

**Comment by Snille** [comment]
Hi, sorry, I don't know french, please translate to English (or Swedish :).

**Comment by jeff44850** [comment]
Hello
First of all thank you for this module since I was looking for a local video broadcaster automatically !!!
However, I have a request on the evolution of this module
Would it be possible to play a set of videos from a folder?
cordially
Jeff

**Comment by Snille** [comment]
Thank you. :) I think there are other "videoplayer-modules" with that feature already. I made this one just for a specific reason really. :) I'm going to use it for showing a specific video in the "fullscreen_below" region when someone have a birthday at home. :) That was the whole idea, and I did not find a player that could be hidden via "notifications" and stopped playing the video when it was hidden... :)
